js/common/modules/@arangodb/aql/explainer.js

```diff
@@ -2628,6 +2654,26 @@ function inspectDump(filename, outfile) {
   });
   print();
 
+  // drop views to be able to recreate analyzers
+  print("/*drop views */");
+  Object.keys(data.views || {}).forEach(function (view) {
+    print("db._dropView(" + JSON.stringify(view) + ");");
+  });
+  print();
+
+  // all analyzers
+  print("/* analyzers setup */");
+  const analyzers_names = Object.keys(data.analyzers);
```

Comment:
This is incompatible with old debug dumps that do not have the `analyzers` flag.